test/unit/testRepeatedTwoPB.js

```diff
@@ -31,13 +33,16 @@ contract('TestRepeatedTwoPB', (accounts) => {
     let bool_field = await contractInstance.getTestRepeatedBool(accounts[0]);
     let unpacked_int32s = await contractInstance.getTestRepeatedUnpackedInt32(accounts[0]);
     let packed_int32s = await contractInstance.getTestRepeatedPackedInt32(accounts[0]);
+    let bzs = await contractInstance.getTestRepeatedBytes(accounts[0]);
 
     assert.equal(string_field_expected, string_field);
     assert.equal(JSON.stringify(uint256s_expected), JSON.stringify(uint256s.map(x => x.toNumber())));
     assert.equal(JSON.stringify(sint64s_expected), JSON.stringify(sint64s.map(x => x.toNumber())));
     assert.equal(bool_field_expected, bool_field);
     assert.equal(JSON.stringify(unpacked_int32s_expected), JSON.stringify(unpacked_int32s.map(x => x.toNumber())));
     assert.equal(JSON.stringify(packed_int32s_expected), JSON.stringify(packed_int32s.map(x => x.toNumber())));
+    assert.equal(JSON.stringify(bzs_expected), JSON.stringify(bzs));
+    console.log(bzs);
```

Comment:
@siburu small question: Is this log an intended change?

Reply:
Thank you for pointing out this. I forgot to delete this.

Reply:
@bluele I've just fixed it now.